rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
```

Comment:
Kindly also elevate the `person` and `book` instances here to accessors so that they have setters and getters.